src/lib.rs

```diff
@@ -98,6 +98,7 @@ pub enum Error {
 /// The number of bytes necessary to store a `FieldElement`.
 const FIELD_BYTES: usize = 32;
 
+#[derive(Debug)]
```

Comment:
In trying to make the `CliError` type generic over a Backend, I was getting an error that the backend didn't implement `Debug` so I derived it here.

This isn't ideal, since we don't require Debug for 0.11 but we may need to require it for 0.12. I'll link the Noir draft once it is done.

Reply:
See https://github.com/noir-lang/noir/pull/1322